### PR TITLE
task 6 complete

### PR DESCRIPTION
**Pull request by FergusJames212**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -120,3 +120,45 @@ describe("GET /api/articles/:id", () => {
...
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number)
+            });
+            expect(comment.article_id).toBe(3);
+          })
+        );
+      });
```
Comment (robert112233) [inline __tests__/app.test.js:144]: Missing test for sorting the comments 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -120,3 +120,45 @@ describe("GET /api/articles/:id", () => {
...
+
+  it("404: returns an error when given a valid article_id but there are no comments on the article", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("No comments on that article");
+      });
+  });
+});
```
Comment (robert112233) [inline __tests__/app.test.js:186]: This is a 200 not an error! If you want to render the comments on a facebook post you won't show an error, you will still just display the comments (0)

File: __tests__/app.test.js [hunk 3]
```diff
@@ -120,3 +120,45 @@ describe("GET /api/articles/:id", () => {
   });
 });
```
Comment (robert112233) [inline __tests__/app.test.js:122]: One missing test, if we request /api/articles/100000/comments it should give you back a 404. You may need a checkExists / Promise.all to differentiate between valid articles with no comments: rows =[] and invalid articles: rows = [] 

This was covered in yesterday's lecture (I think, I wasn't there)

**Review by robert112233 — CHANGES_REQUESTED** [review]
Great start 💯  Amend the tests and submit another PR for review!

**Comment by Harrison365** [comment]
Nice work, Fergus 👍 
Merge away! ✅ 

Harrison :)

**Pull request merged**